README.md

```diff
@@ -1,35 +1,29 @@
-1. [Navigate](#exploring-the-versions-history) to the page with the history of changes you are interested in. _In the OpenTermsArchive example above, this would be [this page](https://github.com/OpenTermsArchive/demo-versions/commits/main/OpenTermsArchive/Privacy%20Policy.md)._
-2. Copy the address of that page from your browser’s address bar. _In the OpenTermsArchive example, this would be `https://github.com/OpenTermsArchive/demo-versions/commits/main/OpenTermsArchive/Privacy%20Policy.md`._
-3. Append `.atom` at the end of this address. _In the OpenTermsArchive example, this would become `https://github.com/OpenTermsArchive/demo-versions/commits/main/OpenTermsArchive/Privacy%20Policy.md.atom`._
-4. Subscribe your RSS feed reader to the resulting address.
+## Usage
 
-#### Recap of available RSS feeds
+🔍 [Browse through versions](https://docs.opentermsarchive.org/how-to/navigate-history/) interactively on this user interface.
 
-| Updated for                         | URL                                                                                                                                                                                            |
-| ----------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| all services and documents          | `https://github.com/OpenTermsArchive/demo-versions/commits.atom`                                                                                                                            |
-| all the documents of a service      | Replace `$serviceId` with the service ID:<br>`https://github.com/OpenTermsArchive/demo-versions/commits/main/$serviceId.atom.`                                                            |
-| One specific document               | Replace `$serviceId` with the service ID and `$documentType` with the document type:<br>`https://github.com/OpenTermsArchive/demo-versions/commits/main/$serviceId/$documentType.md.atom` |
+🔔 [Subscribe to terms changes](https://docs.opentermsarchive.org/how-to/navigate-history/) to be notified.
 
-For example:
-
-- To receive all updates of `Facebook` documents, the URL is `https://github.com/OpenTermsArchive/demo-versions/commits/main/Facebook.atom`.
-- To receive all updates of the `Privacy Policy` from `Google`, the URL is `https://github.com/OpenTermsArchive/demo-versions/commits/main/Google/Privacy%20Policy.md.atom`.
+📥 [Download versions as dataset](https://github.com/OpenTermsArchive/demo-versions/releases) to explore data locally on your filesystem.
 
+If a specific service or specific terms for a service is missing from this collection, it may be available in others [public Open Terms Archive collections](https://github.com/OpenTermsArchive?q=versions&type=all&language=&sort=), and if not, [it can be added](https://docs.opentermsarchive.org/contributing-terms).
```

Comment:
```suggestion
If a specific service or specific terms for a service is missing from this collection, it may be available in others [referenced Open Terms Archive collections](https://opentermsarchive.org/#collections). If not, it can be added by following our documentation about [contributing terms](https://docs.opentermsarchive.org/contributing-terms).
```